tests/helix/send-to-helix-inner.proj

```diff
@@ -60,6 +60,9 @@
 
     <_CleanupProcessesCommand Condition="'$(OS)' == 'Windows_NT'">powershell -ExecutionPolicy ByPass -NoProfile -command "&amp; get-ciminstance win32_process | where-object ExecutablePath -Match 'dotnet-tests|dcp.exe|dcpctrl.exe' | foreach-object { echo $_.ProcessId $_.ExecutablePath %3B stop-process -id $_.ProcessId -force -ErrorAction SilentlyContinue }"</_CleanupProcessesCommand>
```

Comment:
Also here, please do not kill `dcp` and `dcpctrl`. They will eventually exit on their own, promise

Reply:
Discussed offline